documentation/README-Installation-Debian-Buster.txt

```diff
@@ -91,6 +96,7 @@ DENYHOSTS_BIN = "/usr/sbin/denyhosts.py"
 DENYHOSTS_LOCK = "/var/lock/subsys/denyhosts"
 DENYHOSTS_CFG = "/etc/denyhosts.conf"
 
+# The daemon is added in the /etc/init.d folder, but it would seem more coherent to add the denyhosts.service service directly, from this same location.
```

Comment:
@ZerooCool  The service file should be added to /etc/systemd/system/denyhosts.service rather than adding it to /etc/init.d  Adding it to /etc/systemd/system will allow ```service denyhosts start``` to work along with ```systemctl start denyhosts``` to work

Reply:
@josev814 Ok, i have add this. Ok for me, documentation update.

I test later this method, with the branch bug_128.